apprentice_teacher.rb

```diff
@@ -1,52 +1,68 @@
-class ApprenticeTeacher
+require_relative 'senior_teacher'
+require_relative 'module_high_five'
+
+class ApprenticeTeacher < SeniorTeacher
   attr_reader :age, :salary, :phase, :target_raise
   attr_accessor :name
 
-  def initialize(options={})
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
+  
+  RESPONSE = "Listen, class, this is how everything works. *drops crazy knowledge bomb* ... You're welcome."
+  # RESPONSE_RAISE = [Yay, I'm a great employee!", ""Oh, well -- thanks to this actionable, specific, and kind feedback, I'll do better next time."]
+  def initialize(options)
+    super
     @target_raise = 800
-    @phase = 3
+    # @performance_rating = options.fetch(:performance_rating, 0)
+    # # @response2 = 
   end
 
-  def offer_high_five
-    "High five!"
-  end
+  # def offer_high_five
+  #   "High five!"
+  # end
+  include Highfive
 
-  def set_phase(num)
-    @phase = num
-    "Cool, I've always wanted to teach phase #{num}!"
-  end
+  # def set_phase(num)
+  #   @phase = num
+  #   "Cool, I've always wanted to teach phase #{num}!"
+  # end
 
   def teach_stuff
-    response = ""
-    response += "Listen, class, this is how everything works. "
-    response += "*drops crazy knowledge bomb* "
-    response += "... You're welcome."
-    response
+    RESPONSE 
+    # response += "Listen, class, this is how everything works. "
+    # response += "*drops crazy knowledge bomb* "
+    # response += "... You're welcome."
+    # response
   end
 
-  def salary=(new_salary)
-    puts "This better be good!"
-    @salary = new_salary
-  end
+  # def salary=(new_salary)
+  #   puts "This better be good!"
+  #   @salary = new_salary
+  # end
 
-  def receive_raise(raise)
-    @salary += raise
-  end
+  # def receive_raise(raise)
+  #   @salary += raise
+  # end
 
   def set_performance_rating(rating)
```

Comment:
This method is identical to the one in SeniorTeacher except for one thing.  How can we refactor given that?